packages/asset/contracts/CustomSplitter.sol

```diff
@@ -0,0 +1,248 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/AddressUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/proxy/Clones.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@manifoldxyz/royalty-registry-solidity/contracts/libraries/BytesLibrary.sol";
+import {IRoyaltySplitter, IERC165, Recipient} from "@manifoldxyz/royalty-registry-solidity/contracts/overrides/IRoyaltySplitter.sol";
+import {IRoyaltyManager} from "./interfaces/IRoyaltyManager.sol";
+
+interface IERC20Approve {
+    function approve(address spender, uint256 amount) external returns (bool);
+
+    function increaseAllowance(
+        address spender,
+        uint256 amount
+    ) external returns (bool);
+}
+
+/**
+ * Cloneable and configurable royalty splitter contract
+ */
+contract CustomRoyaltySplitter is
```

Comment:
contract has been renamed but not the file
`CustomRoyaltySplitter`

Reply:
and maybe every royalty contracts should start with royalty ?
CustomRoyaltySplitter => RoyaltyCustomSplitter